stack-orchestration/manual-stacks/dns-zones/template.yaml

```diff
@@ -0,0 +1,144 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
For consistency with other template, remove quotation marks? (here and in other locations, notable where there is `!Sub`)